src/System.IO.FileSystem/tests/Tfm.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
```

Comment:
For this one I might add some to the enum in build tools. Some of them are pretty specific to my use case- figured I'd start here and follow up on making things more common.